### PR TITLE
Add doc sub-page "Integrating Scrapy projects"

### PR DESCRIPTION
**Pull request by vdusek**
Closes https://github.com/apify/actor-templates/issues/228

image

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "apify-cli",
-    "version": "0.19.0",
+    "version": "0.19.1",
```
Comment (B4nan) [inline package.json:3]: should be actually 0.19.2 as 0.19.1 is out already (we forgot to bump the version after release again...)

**Review by B4nan — APPROVED** [review]
looking good, we can bump the version after merging as its not related to the PR

File: docs/integrating-scrapy.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+---
+sidebar_label: Integrating Scrapy projects
+title: Integrating Scrapy projects
+---
+
+This part of the documentation describes how to integrate Scrapy projects with Apify platform.
+
+Scrapy, a widely-used open-source web scraping framework, can now be effortlessly migrated to the Apify platform using our dedicated migration tool. This tool empowers users to transform their Scrapy projects into [Apify Actors](https://docs.apify.com/platform/actors) with a single command. To initiate this transformation, execute the following initialization command in the root of your Scrapy project. The command will automatically detect the presence of the `scrapy.cfg` file in your project.
```
Comment (barjin) [inline docs/integrating-scrapy.md:8]:
```suggestion
Scrapy, a widely used open-source web scraping framework, can now be effortlessly migrated to the Apify platform using our dedicated migration tool. This tool empowers users to transform their Scrapy projects into [Apify Actors](https://docs.apify.com/platform/actors) with a single command. To initiate this transformation, execute the following initialization command in the root of your Scrapy project. The command will automatically detect the presence of the `scrapy.cfg` file in your project.
```

File: docs/integrating-scrapy.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
+---
+sidebar_label: Integrating Scrapy projects
+title: Integrating Scrapy projects
+---
+
+This part of the documentation describes how to integrate Scrapy projects with Apify platform.
```
Comment (barjin) [inline docs/integrating-scrapy.md:6]:
```suggestion
This part of the documentation describes how to integrate Scrapy projects with the Apify platform.
```

**Review by barjin — APPROVED** [review]
just some small nits from the [mktg style guide](https://www.notion.so/apify/The-Apify-Writing-Style-Guide-de9fbb99dcd84665b6d3f790fc88b3b6) :)

**Pull request merged**